src/newgrf.cpp

```diff
@@ -66,6 +66,11 @@
 /** List of all loaded GRF files */
 static std::vector<GRFFile *> _grf_files;
 
+std::vector<GRFFile *> GetAllGRFFiles()
+{
```

Comment:
return "const std::vector<>&"?

Reply:
I'm considering just make the variable non-static instead.

Reply:
Another option is to pass the grfid instead of some enumeration of the loaded grfs.
Then it's the same every time.